src/libopensc/reader-pcsc.c

```diff
@@ -2202,7 +2203,8 @@ pcsc_pin_cmd(sc_reader_t *reader, struct sc_pin_cmd_data *data)
 	u8 sbuf[sizeof(PIN_VERIFY_STRUCTURE)>sizeof(PIN_MODIFY_STRUCTURE)?
 		sizeof(PIN_VERIFY_STRUCTURE)+SC_MAX_APDU_BUFFER_SIZE:
 		sizeof(PIN_MODIFY_STRUCTURE)+SC_MAX_APDU_BUFFER_SIZE];
-	size_t rcount = sizeof(rbuf), scount = 0;
+	DWORD rcount = sizeof(rbuf);
```

Comment:
Again, when used as input for pcsc_internal_transmit this potentially leads to the wrong type